windows/src/main/csharp/ch/cyberduck/ui/Views/TransfersItem.xaml

```diff
@@ -0,0 +1,94 @@
+﻿<UserControl x:Class="ch.cyberduck.ui.Views.TransfersItem"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             xmlns:local="clr-namespace:ch.cyberduck.ui.Views"
+             xmlns:viewmodels="clr-namespace:ch.cyberduck.ui.ViewModels"
+             mc:Ignorable="d"
+             d:DesignWidth="450"
+             d:DataContext="{d:DesignInstance Type=viewmodels:TransferViewModel}"
+             TextBlock.FontSize="10">
+    <Grid HorizontalAlignment="Stretch"
+          Margin="0,4">
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="Auto" />
+            <ColumnDefinition Width="*" />
+            <ColumnDefinition Width="Auto" />
+        </Grid.ColumnDefinitions>
+
+        <Image Source="{Binding TransferDirection, Converter={StaticResource TransferDirection}, ConverterParameter=32}"
+               Height="32"
+               Width="32"
+               Margin="7"
+               VerticalAlignment="Top"
+               Grid.Column="0" />
+        <Image Source="{Binding Completed, Converter={StaticResource TransferStatus}, ConverterParameter=32}"
+               Height="32"
+               Width="32"
+               Margin="7"
+               VerticalAlignment="Top"
+               Grid.Column="2" />
+
+        <StackPanel Margin="7,0"
+                    Grid.Column="1">
+            <ComboBox ItemsSource="{Binding Roots}"
+                      SelectedItem="{Binding SelectedTransferItem}"
+                      SelectedIndex="{Binding SelectedTransferItemIndex}"
+                      Style="{StaticResource WhiteComboBoxStyle}"
+                      DisplayMemberPath="Title"
+                      Margin="0,3"
+                      VerticalAlignment="Center"
+                      Grid.Column="1"
+                      Grid.Row="0"
+                      VirtualizingPanel.IsVirtualizing="True"
+                      VirtualizingPanel.VirtualizationMode="Recycling" />
+
+            <Grid Grid.Column="1"
+                  Grid.Row="1">
+                <ProgressBar Value="{Binding Progress}"
```

Comment:
Two controls stacked ontop each other.
ProgressBar shown when Running property is true
TextBlock shown when Running property is false.